packages/relative-values/src/components/View/ListView/index.tsx

```diff
@@ -51,94 +63,132 @@ export const ListView: FC<Props> = ({ model }) => {
-          {() => <AxisMenu axis="rows" sortByAverage={false} />}
-        </DropdownButton>
+  const headerRow = (name: string) => (
+    <CellBox header>
+      <div className="p-1 pt-2 text-sm font-semibold text-slate-600">
+        {name}
       </div>
-      <div
-        className="grid border-r border-b border-gray-200 w-full"
-        style={{
-          gridTemplateColumns: "2fr 2fr 1fr",
-        }}
-      >
-        <CellBox header>
-          <div className="p-2 pt-2 text-sm font-semibold text-slate-600">
-            Name
-          </div>
-        </CellBox>
-        <CellBox header>
-          <div className="p-2 pt-2 text-sm font-semibold text-slate-600">
-            Description
-          </div>
-        </CellBox>
-        <ColumnHeader
-          selectedItem={selectedItem}
-          setSelectedItem={setSelectedItem}
-        />
-        {sortedItems.map((item) => (
-          <Fragment key={item.id}>
-            <CellBox>
-              <div
-                className="p-2 flex justify-end text-slate-700 cursor-pointer font-bold text-sm"
-                onClick={() => setSelectedItem(item)}
-              >
-                <div className="flex-grow text-sm hover:text-slate-900 hover:underline">
-                  {item.name}
-                </div>
-                <div
-                  onClick={() => setSelectedItem(item)}
-                  className={"pl-2 cursor-pointer"}
-                >
-                  <CompassIcon
-                    className={clsx(
-                      "hover:fill-slate-800",
-                      item.id === selectedItem.id
-                        ? "fill-slate-500"
-                        : "fill-slate-200"
-                    )}
-                    size={23}
-                    viewBox={"0 0 25 25"}
-                  />
-                </div>
-              </div>
+    </CellBox>
+  );
 
-              <div className="flex px-2 pb-3">
-                {item.clusterId !== undefined ? (
-                  <div className="flex gap-1 items-center">
-                    <div className="flex-none opacity-50">
-                      <ClusterIcon
-                        cluster={catalog.clusters[item.clusterId]}
-                        selected={true}
+  return (
+    <div>
+      <div className="mb-2 flex">
+        <div className="mr-2">
+          <DropdownButton text="Table Settings">
+            {() => <AxisMenu axis="rows" sortByAverage={false} />}
+          </DropdownButton>
+        </div>
+
+        <input
+          type="text"
+          className="p-1 rounded border border-gray-200 mb-4"
+          defaultValue={search}
+          onChange={(e) => setSearch(e.currentTarget.value)}
+        />
+      </div>
+      <div className={clsx(!!numeratorItem ? "flex" : "auto")}>
+        <div className="flex-2">
+          <div
+            className="grid border-r border-b border-gray-200 w-full"
+            style={{
+              gridTemplateColumns: "2fr 2fr 1fr",
```

Comment:
Is this broken/unfinished?
I see 6 headers, but only 3 columns.
<img width="1493" alt="Captura de pantalla 2023-04-13 a la(s) 14 27 29" src="https://user-images.githubusercontent.com/89368/231875365-033cb92f-c4ea-40d7-8494-8b78e1d7b712.png">